ts/utils/__tests__/url.test.ts

```diff
@@ -26,3 +26,26 @@ describe("getResourceNameFromUrl", () => {
     expect(getResourceNameFromUrl(notAPath)).toEqual(notAPath);
   });
 });
+
+describe("getUrlBasepath", () => {
+  it("should return the same url", () => {
+    const url = "https://www.google.com/it/hello";
+    expect(getUrlBasepath(url)).toEqual(url);
+  });
+
+  it("should return the same url even with params and fragments", () => {
+    const base = "https://www.google.com/it/hello";
+    const suffixes: ReadonlyArray<string> = [
+      "?param1=1&param2=2",
+      "?param1=1&param2=2&params3",
+      "&param1=1?param2=2",
```

Comment:
@pp-ps this case `&param1=1?param2=2` is deliberately absurd (but it could happen, i.e for a bug or an error in the redirect url creation). The test is about stressing the goal of the function: get the basepath of the url excluding all possibile suffixes.

This scenario `%26param1=1` is now covered (good catch!) https://github.com/pagopa/io-app/pull/2076/commits/afaf30164c82ff4f47354e3893def8b8bd3731f5